vitest.setup.ts

```diff
@@ -1,4 +1,10 @@
-import '@testing-library/jest-dom'
-import { config } from 'dotenv-safe'
+import createFetchMock from 'vitest-fetch-mock'
+import { vi } from 'vitest'
+// import { config } from 'dotenv-safe'
 
-config({ path: '.env.test' })
+// config({ path: '.env.test' })
```

Comment:
When will these be uncommented out/removed?

Reply:
When they need to be used, during the rewrite of the signup card functionalities is my guess.

Reply:
Could you add a todo here that mentions this?

Reply:
done.